setup.py

```diff
@@ -0,0 +1,47 @@
+from setuptools import setup
+
+package_name = 'rqt_console'
+
+setup(
+    name=package_name,
+    version='1.1.1',
```

Comment:
Storing the version number here (which is necessary for a Python package) increases the maintenance effort since `catkin_prepare_release` can't be used anymore. I would prefer first having a way to automate this before converting numerous `rqt` plugins to the `ament_python` build type.